k8s/helm/metaflow/charts/metaflow-ui/values.yaml

```diff
@@ -1,55 +1,111 @@
-# Default values for metaflow-ui.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-replicaCount: 1
-
-image:
-  repository: public.ecr.aws/outerbounds/metaflow_metadata_service
-  pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
-
-uiImage:
-  repository: public.ecr.aws/outerbounds/metaflow_ui
-  pullPolicy: IfNotPresent
-  tag: "v1.1.4"
-
+uiBackend:
+  image:
+    name: public.ecr.aws/outerbounds/metaflow_metadata_service
+    pullPolicy: IfNotPresent
+    tag: ""
```

Comment:
Will empty tag pull `lastest` tag ? Should we hardcode something here?

Reply:
This defaults to the chart's appVersion. I will add the comment back.

Reply:
added comment